libtenzir/builtins/operators/python.cpp

```diff
@@ -265,6 +267,13 @@ class python_operator final : public crtp_operator<python_operator> {
       ::close(errpipe.pipe().native_sink());
       co_yield {}; // signal successful startup
       for (auto&& slice : input) {
+        if (!child.running()) {
+          auto python_error = drain_pipe(errpipe);
+          diagnostic::error("{}", python_error)
+            .note("python process exited with exit-code 1")
+            .emit(ctrl.diagnostics());
```

Comment:
Nitpick: You don't know that the exit-code was 1. You can use the actual exit code instead (same below).